content/includes/business-logos.html

```diff
@@ -0,0 +1,21 @@
+<section id="business-logo">
+    <div class="ui very padded container">
+        <h3 class="ui huge center aligned column header">
+            Trusted by businesses worldwide since 2010
+        </h3>
+        <div class="ui padded stackable four column grid">
+            <div class="center aligned column">
```

Comment:
You still want `.middle.aligned` class here, as the images were not resized to the same dimensions. The vertical alignment is off without this:

![image](https://github.com/readthedocs/website/assets/1140183/e9f35abc-abaa-467c-9d05-3e667dcd5588)

```suggestion
            <div class="middle aligned center aligned column">
```